raspberry-pi/4/audio.nix

```
@@ -35,6 +35,12 @@ in
         }
       ];
     };
+
+    # set tsched=0 in pulseaudio config to avoid audio glitches
+    # see https://wiki.archlinux.org/title/PulseAudio/Troubleshooting#Glitches,_skips_or_crackling
+    hardware.pulseaudio.configFile = pkgs.runCommand "default.pa" {} ''
```

```suggestion
    hardware.pulseaudio.configFile = lib.mkDefault pkgs.runCommand "default.pa" {} ''
```

While it won't compose with further changes to `hardware.pulseaudio.configFile` made by the user, at least it won't conflict.